### DISCUSSION TITLE
After initial install of meta from a pax is the meta directory/files still relevant ?

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#689] [General]
After following the install instructions to get started with `zopen` is the `meta-0.8.1` still required or can it be removed?

**Comment by IgorTodorovskiIBM** [comment]
Thanks, I updated the instructions! https://github.com/ZOSOpenTools/meta/releases/tag/v0.8.1 and https://zosopentools.github.io/meta/#/Guides/QuickStart

**Reply by IgorTodorovskiIBM** [reply]
There's also the last line from the zopen init output:
<img width="1253" alt="Screenshot 2024-02-06 at 7 36 22 PM" src="https://github.com/ZOSOpenTools/meta/assets/39890068/3d51773a-23d7-4624-9b58-ff551362c497">

**Comment by lbdyck** [comment]
very nice - thank you
